data/dataGeneratorScripts/generateRenewables/importMarktStammDaten.py

```diff
@@ -0,0 +1,663 @@
+from tkinter.ttk import Separator
+from lxml import etree
+import os
+import re
+import json
+
+class Date():
+
+    @dataclass
+    class Year():
+        year:int
+        def __gt__(self,other) -> bool:
+            returnBool:bool = False
+            if self.year > other.year:
+                returnBool = True
+            return returnBool
+        
+        def __lt__(self,other) -> bool:
+            returnBool:bool = False
+            if self.year < other.year:
+                returnBool = True
+
+            return returnBool
+        
+        def __eq__(self,other) -> bool:
+            returnBool:bool = False
+            if self.year == other.year:
+                returnBool = True
+            return returnBool
+
+        def __ge__(self,other) -> bool:
+            return self > other or self == other
+
+        def __le__(self,other) -> bool:
+            return self < other or self == other
+
+
+    @dataclass
+    class Month():
+        month:int
+        def __gt__(self,other) -> bool:
+            returnBool:bool = False
+            if self.month > other.month:
+                returnBool = True
+            return returnBool
+        
+        def __lt__(self,other) -> bool:
+            returnBool:bool = False
+            if self.month < other.month:
+                returnBool = True
+
+            return returnBool
+        
+        def __eq__(self,other) -> bool:
+            returnBool:bool = False
+            if self.month == other.month:
+                returnBool = True
+            return returnBool
+
+        def __ge__(self,other) -> bool:
+            return self > other or self == other
+
+        def __le__(self,other) -> bool:
+            return self < other or self == other
+
+    @dataclass
+    class Day():
+        day:int
+        def __gt__(self,other) -> bool:
+            returnBool:bool = False
+            if self.day > other.day:
+                returnBool = True
+            return returnBool
+        
+        def __lt__(self,other) -> bool:
+            returnBool:bool = False
+            if self.day < other.day:
+                returnBool = True
+
+            return returnBool
+        
+        def __eq__(self,other) -> bool:
+            returnBool:bool = False
+            if self.day == other.day:
+                returnBool = True
+            return returnBool
+        
+        def __ge__(self,other) -> bool:
+            return self > other or self == other
+
+        def __le__(self,other) -> bool:
+            return self < other or self == other
+
+    year:Year
+    month:Month
+    day:Day
+
+    def __init__(self,year,month,day):
+        self.year = self.Year(year=year)
+        self.month = self.Month(month=month)
+        self.day = self.Day(day=day)
+
+    @classmethod
+    def parse_day_month_year(cls,inputdate:str,separator:str="."):
+        [day,month,year] = inputdate.split(separator)
+        return cls(int(year),int(month),int(day))
```

Comment:
It seems like the built-in [`datetime`](https://docs.python.org/3/library/datetime.html) module can do most of what is needed here.

```python
from datetime import datetime
d1 = datetime.strptime("23.03.2022", "%d.%m.%Y")
print(d1.day, d1.month, d1.year)
# 23 3 2022

d2 = datetime.strptime("01.04.2022", "%d.%m.%Y")
print(d1 < d2)
# True
```